coriolisclient/tests/cli/test_utils.py

```diff
@@ -0,0 +1,310 @@
+
+from coriolisclient.cli import utils
+from coriolisclient.tests import test_base
+
+
+@ddt.ddt
+class UtilsTestCase(test_base.CoriolisBaseTestCase):
+    """Test suite for the Coriolis Utils."""
+
+    def test_add_storage_mappings_arguments_to_parser(self):
+        parser = argparse.ArgumentParser()
+
+        utils.add_storage_mappings_arguments_to_parser(parser)
+        args = parser.parse_args(
+            ['--storage-backend-mapping', 'mock_source=mock_destination',
+             '--disk-storage-mapping', 'mock_disk_id=mock_destination'])
+
+        self.assertEqual(
+            [{'disk_id': 'mock_disk_id', 'destination': 'mock_destination'}],
+            args.disk_storage_mappings
+        )
+        self.assertEqual(
+            [{'source': 'mock_source', 'destination': 'mock_destination'}],
+            args.storage_backend_mappings
+        )
+
+    def test_get_storage_mappings_dict_from_args(self):
+        args = mock.Mock()
+        args.default_storage_backend = mock.sentinel.default_storage_backend
+        args.disk_storage_mappings = mock.sentinel.disk_storage_mappings
+        args.storage_backend_mappings = mock.sentinel.storage_backend_mappings
+
+        result = utils.get_storage_mappings_dict_from_args(args)
+
+        self.assertEqual(
+            {
+                "backend_mappings": mock.sentinel.storage_backend_mappings,
+                "default": mock.sentinel.default_storage_backend,
+                "disk_mappings": mock.sentinel.disk_storage_mappings
+            },
+            result
+        )
+
+    def test_format_mapping(self):
+        mapping = {
+            "mapping1": "mock_mapping1",
+            "mapping2": "mock_mapping2",
+            "mapping3": "mock_mapping3"
+        }
+
+        result = utils.format_mapping(mapping)
+
+        self.assertEqual(
+            (
+                "'mapping1'='mock_mapping1', "
+                "'mapping2'='mock_mapping2', "
+                "'mapping3'='mock_mapping3'"
+            ),
+            result
+        )
+
+    def test_parse_storage_mappings(self):
+        storage_mappings = {
+            "default": "mock_default",
+            "backend_mappings": [
+                {
+                    "source": "mock_source",
+                    "destination": "mock_destination"
+                }
+            ],
+            "disk_mappings": [
+                {
+                    "disk_id": "mock_disk_id",
+                    "destination": "mock_destination"
+                }
+            ]
+        }
+
+        result = utils.parse_storage_mappings(storage_mappings)
+
+        self.assertEqual(
+            (
+                'mock_default',
+                {'mock_source': 'mock_destination'},
+                {'mock_disk_id': 'mock_destination'}
+            ),
+            result
+        )
+
+        result = utils.parse_storage_mappings(None)
+
+        self.assertEqual(
+            (None, {}, {}),
+            result
+        )
```

Comment:
Please move this into a separate test method, call it `test_parse_storageg_mappings_none` or something like that.